### PR TITLE
Created RGB colors.

### PR DESCRIPTION
**Pull request by davidpiuva**
First step of replacing BGR with RGB.

### CODE REVIEW

**Comment by davidpiuva** [comment]
@sebastianbaginski Peer review?

**Comment by marcusnaslund** [comment]
Does not compile.

**Comment by davidpiuva** [comment]
That is because upstream does not compile.

**Comment by davidpiuva** [comment]
Is it safe to fetch again?

File: source/draw/Color.ooc [hunk 1]
```diff
@@ -73,6 +77,8 @@ ColorYuva: cover {
 	toMonochrome: func -> ColorMonochrome { ColorMonochrome new(this y) }
 	toUv: func -> ColorUv { this toYuv() toUv() }
 	toYuv: func -> ColorYuv { ColorYuv new(this y, this u, this v) }
+	toRgb: func -> ColorBgr { this toYuv() toRgb() }
```
Comment (marcusnaslund) [inline source/draw/Color.ooc]: Upstream runs fine. The problem is here. Return type is `Bgr` but you return `Rgb`.

Comment (davidpiuva) [inline source/draw/Color.ooc]: Updated. My compiler did not catch that but something about "CharBuffer lastIndexOf()" is instantly crashing every time.

File: source/draw/Color.ooc [hunk 2]
```diff
@@ -73,6 +79,8 @@ ColorYuva: cover {
 	toMonochrome: func -> ColorMonochrome { ColorMonochrome new(this y) }
 	toUv: func -> ColorUv { this toYuv() toUv() }
 	toYuv: func -> ColorYuv { ColorYuv new(this y, this u, this v) }
+	toRgb: func -> ColorRgb { this toYuv() toRgb() }
+	toRgba: func -> ColorRgba { ColorRgba new(this toRgb(), alpha) }
```
Comment (sebastianbaginski) [inline source/draw/Color.ooc]: `this alpha`
`this` is missing in `toBgra` too.

**Comment by sebastianbaginski** [comment]
Looks good.

File: source/draw/Color.ooc (hunk 2 above)
Comment (davidpiuva) [inline source/draw/Color.ooc]: Updated

**Pull request merged**